### PR TITLE
55 a landing page

### PR DESCRIPTION
**Pull request by NLMOTS**

### CODE REVIEW

File: frontend/src/app/modules/user-card/user-card.component.html [hunk 1]
```diff
@@ -3,30 +3,30 @@
         <div class="user-general-info">
             <div class="left-area">
                 <div class="picture">
-                    <span class="picture-placeholder">Photo</span>
+                    <!-- <span class="picture-placeholder">Photo</span> -->
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/user-card/user-card.component.html:6]: Remove comment

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by SidletskyiOlexandr — DISMISSED** [review]

File: frontend/src/app/modules/landing/landing-page/landing-page.component.sass [hunk 2]
```diff
@@ -0,0 +1,470 @@
+@import "src/styles/colors"
+
+@media screen and (min-width: 2540px)
+  .all
+    button
+        h3
```
Comment (tatianahutii) [inline frontend/src/app/modules/landing/landing-page/landing-page.component.sass:6]: I believe we don't need so many new styles

File: frontend/src/app/shared/utils/user-card.utils.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export class Utils {
 
         language: 'English',
 
-        imagePath: 'assets/user-card-icons/Photo.png',
+        imagePath: '../../../assets/user-card-icons/Photo.png',
```
Comment (tatianahutii) [inline frontend/src/app/shared/utils/user-card.utils.ts:11]: why did you change this?

File: frontend/src/assets/topic-icons/Alarm Clock.svg
Comment (tatianahutii) [inline frontend/src/assets/topic-icons/Alarm Clock.svg:1]: names without space
everywhere

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/landing/landing-page/landing-page.component.sass (hunk 2 above)
Comment (NLMOTS) [reply frontend/src/app/modules/landing/landing-page/landing-page.component.sass:6]: maybe i need to split landing page in few component, to make it easier to read, but i don't think that i could decrease variable amount for easy

File: frontend/src/styles/colors.sass [hunk 4]
```diff
@@ -2,6 +2,9 @@ $pink-100: #F8D5E8
 $white: #FFFFFF
 $turquoise-100: #61D4F6
 $blue-200: #1C2FBA
+$blue-100: #E0E2F2
```
Comment (Anatolii-Ivanov) [inline frontend/src/styles/colors.sass:5]: We have this color, so either change everywhere either use existing

Comment (NLMOTS) [reply frontend/src/styles/colors.sass:5]: My bad, now i will check if color already exist

**Review by Anatolii-Ivanov — APPROVED** [review]
